### DISCUSSION TITLE
Add custom status for a ticket

### DISCUSSION THREAD

**Discussion by gmarcostam** [polonel/trudesk#482] [Ideas]
Hi,

I'm wordering if it is possibile to add a section in settings where it could be possibile to add custom status for a ticket
Sometimes we need to classify a ticket in a status which is not listed in the default status:
- New
- Pending
- Open
- Close 

Maybe it would be the case to get the status list from a mongodb collection.

Do you think it is affordable as request in a future release?

Thanks for your work

**Comment by polonel** [comment]
This has been planned for a while but would require many rewrites as a lot of the status were hard-coded originally. It is something that is still on the backlog and will make it in eventually.
